### PR TITLE
MIDI support

### PR DESCRIPTION
**Pull request by Unrud**
Install **fluidsynth** and **Fluid** soundfont for MIDI support.

Use wrapper for **scummvm**, to set ``music_driver`` and ``soundfont`` in configuration.

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Thanks for the PR! I'll review it when I get home and hopefully merge it soon.

**Comment by AsciiWolf** [comment]
bot, build

File: org.scummvm.ScummVM.json [hunk 1]
```diff
@@ -32,11 +71,22 @@
...
                     "type": "file",
                     "path": "org.scummvm.ScummVM.appdata.xml"
                 }
             ],
+            "config-opts": [
+                "--enable-release",
+                "--with-fluidsynth-prefix=/app"
+            ],
+            "build-commands": [
+                "sed -i s:Exec=scummvm:Exec=/app/bin/scummvm_wrapper: dists/scummvm.desktop"
```
Comment (AsciiWolf) [inline org.scummvm.ScummVM.json]: `Exec=scummvm_wrapper` should be sufficient. The `/app/bin` directory is already in PATH.

File: org.scummvm.ScummVM.json [hunk 2]
```diff
@@ -24,6 +24,45 @@
...
+        {
+            "name": "fluidsynth",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FluidSynth/fluidsynth/archive/v2.0.0.tar.gz",
+                    "sha256": "5ca094efbe13fdb880cfd0431354a7b85daf7239c344939493a2aaeca4e59ed5"
+                }
+            ],
+            "buildsystem": "cmake",
```
Comment (TingPing) [inline org.scummvm.ScummVM.json]: `cmake-ninja`

File: org.scummvm.ScummVM.json [hunk 3]
```diff
@@ -24,6 +24,45 @@
...
+            "config-opts": [
+                "-DLIB_INSTALL_DIR=lib"
```
Comment (TingPing) [inline org.scummvm.ScummVM.json]: Shouldn't be needed.

File: org.scummvm.ScummVM.json [hunk 4]
```diff
@@ -24,6 +24,45 @@
...
+            ]
+        },
+        {
+            "name": "fluid_soundfont",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://http.debian.net/debian/pool/main/f/fluid-soundfont/fluid-soundfont_3.1.orig.tar.gz",
```
Comment (TingPing) [inline org.scummvm.ScummVM.json]: Is the real upstream unreliable or something?

Comment (Unrud) [reply org.scummvm.ScummVM.json]: I couldn't find the real upstream. Debian downloaded the file from MuseScore, but the download link is gone. MuseScore still has files (with newer file formats) that are based on it.

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
LGTM, but I'll keep this open for a while.

**Pull request merged**